zip-0227.rst

```diff
@@ -263,6 +275,7 @@ For each `IssueAction` in `IssueBundle`:
 
 - check that :math:`0 < \mathsf{assetDescSize} <= 512`.
 - check that :math:`\mathsf{asset\_desc}` is a string of length :math:`\mathsf{assetDescSize}` bytes.
+
```

Comment:
`asset_desc` is defined as "UTF-8 encoded string, of size assetDescSize bytes" in the table. Yet, it's Byte size is set to `Varies` and its type is set as `byte`. 

It seems like it's Byte size shouldn't be `Varies` but `assetDescSize` instead and it's type should be a byte array `byte[assetDescSize]`